aries-site/src/pages/templates/content-layouts.mdx

```diff
@@ -32,39 +71,89 @@ Specifying layout content in this way will also ensure that it is interepreted a
+regions in which the content will live, whether those regions are [fixed, flexible, or fluid](/components/grid#background), 
+and how those regions are [prioritized](#prioritizing-content-needs) at each responsive breakpoint.
+
+Container-driven layouts are useful for defining larger layouts, such as at the page level, 
+but can be useful for defining smaller sub-regions as well. The best use of available screen 
+real-estate can be achieved by combining a container-driven page layout with things like Cards
+which have been built to have adaptive layouts themselves.
+
+The following example takes a container-driven approach, implementing a an app-catalog 
+consisting of a filters region and a results region.
+
+Key aspects to the approach are:
+- At larger breakpoints, the content layout will be a two-column layout; a single column 
+at smaller breakpoints.
+- The filters region will be fixed-width when in a two-column layout and [flex-width](/components/grid#flexible-columns) 
+when in a single column. The results region always uses a [fluid grid](/components/grid#fluid-grids).
+- The filters region are prioritized over the results region because the filters are needed to control 
+the results.
+
+<Example
+  showResponsiveControls={['fullScreen']}
+  code={[
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/ContainerDrivenLayout/ContainerDrivenLayout.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/PageHeader/PageHeader.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/ContainerDrivenLayout/AppResults.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/ContainerDrivenLayout/FilterPanel.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/data/mockData/applications.json",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/ContainerDrivenLayout/constants.js",
+    ]}
+  github="https://github.com/grommet/hpe-design-system/blob/master/aries-site/src/examples/templates/content-layouts/ContainerDrivenLayout.js"
+  // height, width, and screenContainer.scale value chosen to best visually 
+  // represent the example
+  height="650px"
+  width="100%"
+  screenContainer={{ scale: 0.75 }}
+>
+  <ContainerDrivenLayout />
+</Example>
 
-See the table below for breakpoints defined by the HPE Design System. Keep in mind that these breakpoints are not meant to target every single existing device &mdash; we understand that technology is ever evolving. Instead, the ranges defined here help build a strong foundation to account for diverse use cases when creating responsive designs.
+### Content-Driven Layouts
```

Comment:
Can we create a ticket to add an example for this layout type (maybe just updating one of our modal examples to use this approach and then add it here?

Don't feel like we have to hold up this PR just to add an example.